pkgs/applications/misc/dbeaver/default.nix

```diff
@@ -62,6 +53,12 @@ stdenv.mkDerivation rec {
     outputHash = "sha256-xKlFFQXd2U513KZKQa7ttSFNX2gxVr9hNsvyaoN/rEE=";
```

Comment:
That probably requires an update, too.

Reply:
hmm, i did set both sha256's to an invalid checksum but this one does not change. Will investigate that tonight.

Reply:
I ran the `buildPhase` inside two clean VMs (one with 21.0.0 and one with 21.0.1 checked out). 

The only difference in the resulting .m2 directories are the `_remote.repositories` files which are cleaned by the `installPhase` and contain the current timestamp/datetime.

Reply:
Then they did not change deps.